CHANGES.md

```diff
@@ -1,4 +1,14 @@
+## 0.200.2 (2023-05-08)
+
+- [UPGRADED] Update SonarQube to support Gradle 8 (#158)
+- [UPGRADED] Switch from GPG to CISO signing solution (#161)
+- [UPGRADED] Bump com.ibm.cloud:cloudant from 0.4.3 to 0.5.1 (#167)
+- [UPGRADED] Bump junit:junit from 4.12 to 4.13.2 (#166)
+- [UPGRADED] Bump org.powermock:powermock-api-easymock (#168)
+- [UPGRADED] Bump org.easymock:easymock from 3.4 to 4.3 (#170)
```

Comment:
I don't think we need to list all of these as the automatically generated release notes will do that.

I think we probably only need to call out `com.ibm.cloud:cloudant from 0.4.3 to 0.5.1` as that is the only one that is likely to have any relevance to end users.

Reply:
tbh I'd prefer changelog to be even more automatic rather than arbitrary handcrafted, but sure. removed in https://github.com/IBM/cloudant-kafka-connector/pull/171/commits/32d1c68af9c680e500bffea9c3f7ac2e5bbf5f56